src/domain/challenge/challenge.resolver.fields.ts

```diff
@@ -51,11 +47,7 @@ export class ChallengeResolverFields {
     return opportunities;
   }
 
-  @Roles(
-    RestrictedGroupNames.CommunityAdmins,
-    RestrictedGroupNames.EcoverseAdmins,
-    RestrictedGroupNames.Members
-  )
+  @Roles(RestrictedGroupNames.Members)
```

Comment:
This will restrict only to the Members, not all levels. Roles need to be added explicitly.